utils.sh

```diff
@@ -61,6 +61,15 @@ function create_cluster() {
         cp assets/ipv6-dual-stack-no-upgrade.yaml ${assets_dir}/openshift/.
     fi
 
+    if [[  ! -z "${ENABLE_CBO_TEST}" ]]; then
```

Comment:
Does anything look at the images used by the deployment before a pod is created? I don't think it's a problem to create this image, I just wonder if we actually need it.

Reply:
Just wanted to have a fixed simple image to reference in case the pod of the deployment is going to go up (even though now I changed the replicas to 0) - the image spec is anyhow required

Reply:
OK, let's keep this as it is, then.